mxenabled/mx-design-tokens
# Aw/ct 210

#### Issue:
https://mxcom.atlassian.net/browse/CT-210

#### Changes
Changed the color of the focus ring to comply with ADA standards it will now have a 3:1 contrast ratio

#### To Test: 

link this repo to the kyper-react repo and do a `npm run dev` to build kyper locally. Search for button and make sure the focus color matches the one in the token.
